cli/clipper/clipper.go

```diff
@@ -0,0 +1,113 @@
+package clipper
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/atotto/clipboard"
+	"github.com/supitsdu/clipper/cli/options"
+)
+
+// ContentReader defines an interface for reading content from various sources.
+type ContentReader interface {
+	Read() (string, error)
+}
+
+// ClipboardWriter defines an interface for writing content to the clipboard.
+type ClipboardWriter interface {
+	Write(content string) error
+}
+
+// FileContentReader reads content from a specified file path.
+type FileContentReader struct {
+	FilePath string
+}
+
+// Read reads the content from the file specified in FileContentReader.
+func (f FileContentReader) Read() (string, error) {
+	content, err := os.ReadFile(f.FilePath)
+	if err != nil {
+		return "", fmt.Errorf("error reading file '%s': %v", f.FilePath, err)

```

Comment:
You are not returning sentinel error, please let tge people using your code be able to catch the ones returned by io.ReadFile

Let's wrap the error

```suggestion
		return "", fmt.Errorf("error reading file '%s': %w", f.FilePath, err)
```